ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml

```diff
@@ -321,7 +321,52 @@
 		<ns1:Description>[In Body] For more details about the airsyncbase:Body element, see [MS-ASAIRS] section 2.2.2.4.</ns1:Description>
 		<ns1:Behavior>Protocol</ns1:Behavior>
 		<ns1:Scope>Both</ns1:Scope>
-		<ns1:IsNormative>Informative</ns1:IsNormative>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
+	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASNOTE_R43001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.1</ns1:Doc_Sect>
+		<ns1:Description>[In Body] Protocol version 12.1 does not supports this [In Body] element.</ns1:Description>
```

Comment:
"this" indicates "Body" but not "In Body". So please update it to:
[In Body] Protocol version 12.1 does not supports this [Body] element.
Please also update the similar issues.